packages/components/src/__future__/Tag/Tag/_docs/TagMigration.stories.tsx

```diff
@@ -0,0 +1,321 @@
+      </Tag>
+      <Tag classNameOverride="ps-4">
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" fullName="Reed Richards" />
+          Reed Richards
+        </span>
+      </Tag>
+      <Tag classNameOverride="ps-4">
+        <span className="flex gap-6 items-center">
+          <Avatar
+            size="small"
+            fullName="Sue Storm"
+            avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+          />
+          Susan Storm
+        </span>
+      </Tag>
+    </div>
+  ),
+}
+
+export const AvatarRemovableMigration: StoryObj = {
+  render: () => (
+    <div className="flex gap-12">
+      <RemovableTag
+        removeButtonProps={{
+          ariaLabel: "Remove user from *this context*",
+          onClick: fn(),
+        }}
+        classNameOverride="ps-4"
+      >
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" />
+          Tag
+        </span>
+      </RemovableTag>
+      <RemovableTag
+        removeButtonProps={{
+          ariaLabel: "Remove Reed Richards from *this context*",
+          onClick: fn(),
+        }}
+        classNameOverride="ps-4"
+      >
+        <span className="flex gap-6 items-center">
+          <Avatar size="small" fullName="Reed Richards" />
+          Tag
+        </span>
+      </RemovableTag>
+      <RemovableTag
+        removeButtonProps={{
+          ariaLabel: "Remove Sue Storm from *this context*",
+          onClick: fn(),
+        }}
+        classNameOverride="ps-4"
+      >
+        <span className="flex gap-6 items-center">
+          <Avatar
+            size="small"
+            fullName="Sue Storm"
+            avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+          />
+          Tag
+        </span>
+      </RemovableTag>
+    </div>
+  ),
+}
+
+export const InlineMigration: StoryObj = {
+  render: () => (
+    <>
+      <div className="mb-6">
+        <Tag>Tag</Tag>
+        <Tag>Tag</Tag>
+        <Tag>Tag</Tag>
+      </div>
+      <div className="flex gap-12 mb-6">
+        <Tag>Tag</Tag>
+        <Tag>Tag</Tag>
+        <Tag>Tag</Tag>
+      </div>
+      <div>
+        <Tag classNameOverride=" me-12">Tag</Tag>
+        <Tag classNameOverride=" me-12">Tag</Tag>
+        <Tag>Tag</Tag>
+      </div>
+    </>
+  ),
+}
+
+export const TruncateMigration: StoryObj = {
+  render: () => (
+    <>
+      <div className="mb-6">
+        <Tag>
+          <span className="max-w-[200px] overflow-ellipsis overflow-x-hidden whitespace-nowrap">
+            I am a really long tag that needs to be shortened
+          </span>
+        </Tag>
+      </div>
+    </>
+  ),
+}
+
+export const SizesMigration: StoryObj = {
+  render: () => <Tag>Tag</Tag>,
+}
```

Comment:
Don't forget that StoryObj is usually used like this :)

```suggestion
export const SizesMigration: StoryObj = {
  args: {
    children: "Tag"
  }
}
```

Or if `children` is pre-defined in the Meta, it can just be this:
```suggestion
export const SizesMigration: StoryObj = {}
```

Reply:
Yeah, always forget about that one ^ will update 👍